portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -1,6 +1,6 @@
 definition {
 
-	@summary = "Default summary"
+	@summary = "This will get the company log content for ${groupId}"
```

Comment:
"This will assert ${logContent} for company ${portalInstanceName} contains the unique group ID logging"